articles/cognitive-services/Bing-Web-Search/quickstarts/nodejs.md

```diff
@@ -22,102 +22,145 @@ Use this quickstart to make your first call to the Bing Web Search API and recei
 Here are a few things that you'll need before running this quickstart:
 
 * [Node.js 6](https://nodejs.org/en/download/) or later
-* A subscription key  
+* A subscription key
 
 ## Create a project and declare required modules
 
-Create a new Node.js project in your favorite IDE or editor. Then copy the code snippet below into your project. This quickstart uses strict mode and requires the `https` module to send and receive data.
+Create a new Node.js project in your favorite IDE or editor.
+Then copy the code snippet below into your project in a file named `search.js`.
+First we require `https` from the standard library, we'll use it to send and receive data.
 
 ```javascript
-// Use strict mode.
-'use strict';
+// Usage:
+//
+//   node search.js "My awesome query"
+//
+const https = require('https')
+```
+
+## Set the subscription key
+
+This code looks up the environment variable, `AZURE_SUBSCRIPTION_KEY`,
+which you can find on
+[your apis page](https://azure.microsoft.com/en-us/try/cognitive-services/my-apis/?apiSlug=search-api-v7).
 
-// Require the https module.
-let https = require('https');
+```javascript
+// HOWTO: set an environment variable https://is.gd/zu55jA
```

Comment:
This is a gap in our documentation, which we plan to address, however, we need to remove this reference to the Twilio documentation.